Client/mods/deathmatch/logic/CClientWebBrowser.cpp

```diff
@@ -305,6 +305,16 @@ void CClientWebBrowser::Events_OnAjaxRequest(CAjaxResourceHandlerInterface* pHan
     pHandler->SetResponse(result);
 }
 
+void CClientWebBrowser::Events_OnConsoleMessage(const SString& strMessage, const SString& strSource, int line, short level)
+{
+    CLuaArguments Arguments;
+    Arguments.PushString(strMessage);
+    Arguments.PushString(strSource);
+    Arguments.PushNumber(line);
+    Arguments.PushNumber(level);
+    CallEvent("onClientBrowserConsoleMessage", Arguments, false);
```

Comment:
camelCase

Reply:
above, everywhere is used like this, it's not problem that needs to be fixed to get merged, if you really want to change things like this just create refactor pull request.

Reply:
New PRs shouldn't introduce code that requires a refactor later to meet current standards. Could go even deeper, hungarian notation shouldn't be used anymore.